kotlinpoet/src/commonMain/kotlin/com/squareup/kotlinpoet/jvm/alias/JvmClass.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2024 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.kotlinpoet.jvm.alias
+
+import kotlin.reflect.KClass
+
+/**
+ * An expected typealias for `java.lang.Class`.
+ */
+public expect class JvmClass<T : Any>
+
+/**
+ * Convert [JvmClass] to [KClass].
+ */
+public expect val <T : Any> JvmClass<T>.kotlin: KClass<T>
```

Comment:
Should we make this `val` internal? It shouldn't really be used outside of the library.

Reply:
Indeed, it doesn't seem to be used outside. I would change it to `internal` .